src/inventory/handler.py

```diff
@@ -2,13 +2,15 @@
 # License:
 # This sample code is made available under the MIT-0 license. See the LICENSE file.
 from inventory.readers import AwsConfigInventoryReader
+from inventory.reports import send_email_with_attachments
 from inventory.reports import CreateReportCommandHandler, DeliverReportCommandHandler
 
 def lambda_handler(event, context):
     inventory = AwsConfigInventoryReader(lambda_context=context).get_resources_from_all_accounts()
 
     report_path = CreateReportCommandHandler().execute(inventory)
     report_url = DeliverReportCommandHandler().execute(report_path)
+    send_email_with_attachments(report_path)
```

Comment:
I added this functionality in `send_email_with_attachments` function. Check the commit https://github.com/Alpacked/fedramp-integrated-inventory-workbook/pull/1/commits/5a012255263dd3cef885b3df92486af1576d2cce

Reply:
Why do you need to execute a function to skip the rest of its body instead of not executing a function? This is misleading and doesn't make sense. The logic should be simple and straightforward - you don't need to send an email -> you don't execute send_email function

Reply:
Yes, agree with you. It really makes sence